flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -154,6 +203,11 @@ class FlxGamepad implements IFlxDestroyable
 		return gamepadButton;
 	}
 	
+	public inline function getFlipAxis(AxisID:Int):Int
```

Comment:
Ah, right, I didn't remember that correctly. That's what the old code did as well actually.

```haxe
if (model == XBox360 && !_isChrome)
	axisValue = -axisValue;
```